preditor/gui/workbox_mixin.py

```diff
@@ -26,6 +64,17 @@ def __init__(
         self._tempfile = tempfile
         self.core_name = core_name
 
+        self._tab_widget = parent
+
+    def __tab_widget__(self):
+        """Return the tab widget which contains this workbox
+
+        Returns:
+            self._tab_widget (GroupedTabWidget): The tab widget which contains
+                this workbox
```

Comment:
I would drop the implementation detail from the docstring. Ie don't specify `self._tab_widget` in the docstring. Other implementations might use a different variable etc,  and hopefully plugins won't be using the private variable.

```
       Returns:
            GroupedTabWidget: The tab widget which contains this workbox
```